src/Dapr.AspNetCore/DaprEndpointRouteBuilderExtensions.cs

```diff
@@ -55,7 +55,7 @@ public static IEndpointConventionBuilder MapSubscribeHandler(this IEndpointRoute
                     {
                         if (logger != null)
                         {
-                            logger.LogWarning("Topic subscription doesn't support route with parameters. Subscription for topic {name} is removed.", entry.Name);
+                            logger.LogInformation("Topic subscription doesn't support route with parameters. Subscription for topic {name} is removed.", entry.Name);
```

Comment:
keys for logging are usually PascalCase. I'd suggest something less generic like `TopicName`.

Reply:
Also I think error is correct here. This is a programming mistake, something you tried to do will not work.